script.js

```diff
@@ -0,0 +1,125 @@
+const API_KEY = '7a7de07845bd21df37d2fed08dc1f49c';
+const currstate = {
+  page: 1,
+  searchTerm: ""
+};
+
+const searchForm = document.getElementById("search-form");
+const searchInput = document.getElementById("bar");
+const resultsEl = document.getElementById("results");
+
+const rootUrl = 'https://api.themoviedb.org/3/discover/movie?include_adult=false&include_video=false&language=en-US&sort_by=popularity.desc&page=';
+const filterUrl = 'https://api.themoviedb.org/3/search/movie?include_adult=false&language=en-US&page=1';
+const apiKey = API_KEY;
+
```

Comment:
good job declaring global variables at the top of the file (and using `const` appropriately)